### PR TITLE
My test has been done

### PR DESCRIPTION
**Pull request by Victoria-Fire**

### CODE REVIEW

File: Hero/Super_Hero/Superman.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+system.out.println ("Привет)
+```
+
+Hi friend
+
+```java
+public int piterson(int a){
+smb
+}
+```
```
Comment (Superman-777) [inline Hero/Super_Hero/Superman.md:19]: Deleted it.

**Review by Superman-777 — CHANGES_REQUESTED** [review]
See notes

- [ ] Are you understand?

File: Hero/Super_Hero/Superman.md (hunk 1 above)
Comment (Victoria-Fire) [reply Hero/Super_Hero/Superman.md:19]: Ок

File: Hero/Super_Hero/Superman.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+I am Superman. I will save your life.
+
+```
+Test code
+```
+
+```java
+system.out.println ("Привет)
+```
```
Comment (Superman-777) [inline Hero/Super_Hero/Superman.md:11]: Good job

**Review by Superman-777 — APPROVED** [review]
lgfm

**Pull request merged**